storybookjs/storybook
# Addon-toolbars: Add optional label for toolbar items

Issue: #14187

## What I did
I added a condition to check if a `showName` is defined and true and display the name of the custom toolbar item next to where the icon is displayed in the toolbar manager.

## How to test
A very straightforward way would be to add `showName: true` where the item is defined in globalTypes and then check if the name is displayed correctly. I used the `official-storybook` example to test the display of the label. Maybe we should add it as a default in other examples as well.

- Is this testable with Jest or Chromatic screenshots?

I am not sure.

- Does this need a new example in the kitchen sink apps?

Maybe we could update the examples to show the label feature. I have updated the `official-storybook` example to include `showName: true` for the `Theme` item. The commit is included in this Pull Request.

- Does this need an update to the documentation?

I think it would be good to add the new feature in the documentation, but I am not sure about where is the exact place it should be added. I could make any updates needed.
